frontend/src/pages/ReadingOlympics/ROHome.js

```diff
@@ -0,0 +1,147 @@
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import React, { Component } from 'react';
+import { Container, Col, Button, Card, CardBody, CardTitle } from 'reactstrap';
+import PReadsNavbar from '../../components/PReadsNavbar';
+import Homefeed from '../../components/Homefeed';
+import Exercises from '../../components/Exercises';
+import Readings from '../../components/Readings';
+import Games from '../../components/Games';
+import Results from '../../components/Results';
+import '../../styles/ReadingOlympics.scss';
+import { getROArchiveYears } from '../../utils/api';
+
+const pathToMiddle = 'middle';
+const pathToIntermediate = 'intermediate';
```

Comment:
what is the difference between middle and intermediate?

Reply:
resolved offline --- Reading Olympics has 2 categories -- intermediate, and middle school